Master Fails to Compile Android

./../../libs/openFrameworks/sound/ofSoundStream.h:14:31: error: cannot allocate an object of abstract type 'ofxAndroidSoundStream'
  #define OF_SOUND_STREAM_TYPE ofxAndroidSoundStream

@arturoc can you build?
I'm trying out a fresh bring up of Eclipse.

well now I get this when I try to build an project

../../../libs/FreeImage/lib/android/armeabi-v7a/libfreeimage.a(libraw_cxx.o):libraw_cxx.cpp:function LibRaw_bigfile_datastream::get_char(): error: undefined reference to '__srget'
collect2: error: ld returned 1 exit status
make[1]: *** [libs/armeabi-v7a/libOFAndroidApp.so] Error 1
make: *** [Release] Error 2

--- a/addons/ofxAndroid/src/ofxAndroidSoundStream.h
+++ b/addons/ofxAndroid/src/ofxAndroidSoundStream.h
@@ -10,7 +10,7 @@
 class ofxAndroidSoundStream : public ofBaseSoundStream{
 	public:
 		ofxAndroidSoundStream();
-		virtual ~ofxAndroidSoundStream();
+        ~ofxAndroidSoundStream();
 		
 		void listDevices();
 		void setDeviceID(int deviceID);
@@ -26,6 +26,7 @@ class ofxAndroidSoundStream : public ofBaseSoundStream{
 		
 		long unsigned long getTickCount();		
 
+        int getDeviceID(){return 0;}
 		int getNumInputChannels();
 		int getNumOutputChannels();
 		int getSampleRate();

